### PR TITLE
XML code blocks 

### PR DESCRIPTION
**Pull request by cloudn9**
Continues list of #9744

### CODE REVIEW

File: docs/framework/wcf/troubleshooting-the-getting-started-tutorial.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ To turn off security:
 - For the client, in the configuration file, update the **\<security>** element under the **\<binding>** element as follows:  
   
     ```xml
-    <binding name="WSHttpBinding_ICalculator" security mode="None" />
+    <binding name="WSHttpBinding_ICalculator" security="None" />
```
Comment (gewarren) [inline docs/framework/wcf/troubleshooting-the-getting-started-tutorial.md]:
```suggestion
    <binding name="WSHttpBinding_ICalculator">
       <security mode="None" />
    <binding />
```

See https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/wshttpbinding for reference.

File: docs/framework/windows-workflow-foundation/flowchart-workflows.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ To model a basic sequence in which two activities execute in turn, the `FlowStep
     </Assign>
     <FlowStep.Next>
       <FlowStep>
-        <WriteLine Text="["Hello, " & result]"/>
+        <WriteLine Text="Hello, & [result]"/>
```
Comment (gewarren) [inline docs/framework/windows-workflow-foundation/flowchart-workflows.md]: Or should it be:

```suggestion
        <WriteLine Text="Hello, " & [result]/>
```

**Review by gewarren — COMMENTED** [review]
I think a couple things need to be fixed here.

File: docs/framework/windows-workflow-foundation/flowchart-workflows.md (hunk 2 above)
Comment (cloudn9) [reply docs/framework/windows-workflow-foundation/flowchart-workflows.md]: Oh I had severe doubts about this one :|..I will correct it right away. Thank you @gewarren

**Review by gewarren — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**